english_tasks/irregular_verbs.py

```diff
@@ -74,52 +115,46 @@ def display_statistic(list1, list2, list3):
     print(f'Fixed verbs:   ', end="")
     for i in list3:
         print(i, end=",")
+    print(f'\nYour scores is {scores}%')
```

Comment:
- declare new variable `result_dict = {"Correct Verbs": <correct_verbs_list>, "Wrong Verbs": <Wrong_verbs_list>, "Fixed Verbs": <fixed_verbs_list>, }`
- maybe use `result.items()` in for loop